.ci/convert_tap_to_junit.sh

```diff
@@ -0,0 +1,21 @@
+##!/usr/bin/env bash
+HOME=/app
+rm -fr node_modules
+npm i tap-junit
+
+for tf in $(ls *-output.tap)
+do
+  filename=$(basename ${tf} output.tap) 
+  if [ -s ${tf} ]; then
+    cat ${tf}|./node_modules/.bin/tap-junit --package="Agent Node.js" > junit-${filename}-report.xml || true
```

Comment:
An alternative to the `npm i tap-junit` above and using the `node_modules/.bin` path here would to just do `npx tap-junit`. What do you think, @watson?

Reply:
I can add node_modules/.bin to the PATH also

Reply:
I don't mind calling `./node_modules/.bin` like this, but I'm a bit unsure if we can use `npx` as it might try to download a newer version of the module? I think adding it to the `PATH` would be a bit "magic", so it's harder to see where it's getting it from?